### PR TITLE
Landing page links

### LINKED ISSUE #64: Landing Page Links to other pages

### PR DESCRIPTION
**Pull request by ChuckBTaylor**
closes #64

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,10 +24,16 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
       rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oswald&display=swap"
+      rel="stylesheet"
+    />
```
Comment (ChuckBTaylor) [inline public/index.html:36]: New font style

File: src/index.css [hunk 2]
```diff
@@ -5,6 +5,9 @@ body {
   -moz-osx-font-smoothing: grayscale;
   background-color: #212529;
 }
+html {
+  font-size: 16px;
```
Comment (ChuckBTaylor) [inline src/index.css:9]: base size for `rem` font sizes

File: src/theme.tsx [hunk 3]
```diff
@@ -1,95 +1,111 @@
-import { createTheme, responsiveFontSizes } from '@mui/material'
+import { createTheme } from '@mui/material'
 
-export const kurtTheme = responsiveFontSizes(
```
Comment (ChuckBTaylor) [inline src/theme.tsx:3]: Removed `responsiveFontSizes

File: src/theme.tsx [hunk 4]
```diff
@@ -1,95 +1,111 @@
...
-        main: '#9EEAF9',
-      },
-      series: { main: '#F7D6E6' },
-      animation: { main: '#A3CFBB' },
-      documentary: { main: '#FECBA1' },
-      background: {
-        default: '#343A40',
+    fontFamily: ['Roboto', 'sans-serif'].join(','),
+    h1: {
+      fontSize: '1.75rem',
```
Comment (ChuckBTaylor) [inline src/theme.tsx:10]: Updated fontSizes to `rem`. We should use the fontSize in Figma  divided by the root html font size (current 16)

File: src/components/organisms/landingPageLinks/LandingPageLinks.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import { EnumImageListItem } from '../../molecules/enumImageListItem/EnumImageListItem'
+
+type LandingPageLinksProps = {
+  pages: PageEnum[]
+}
+
+export function LandingPageLinks({ pages }: LandingPageLinksProps) {
+  const theme = useTheme()
+  const matches = useMediaQuery(theme.breakpoints.up('md'))
+  theme.breakpoints.up('md')
```
Comment (mwilkins74) [inline src/components/organisms/landingPageLinks/LandingPageLinks.tsx:12]: This line looks redundant to me. Remove?

File: src/components/organisms/landingPageLinks/LandingPageLinks.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <ImageList cols={matches ? 2 : 1} gap={0} sx={{ margin: '0px' }}>
+      {pages.map(pageEnum => (
+        <EnumImageListItem pageEnum={pageEnum} key={`${pageEnum}-image-link`} />
+      ))}
+    </ImageList>
```
Comment (mwilkins74) [inline src/components/organisms/landingPageLinks/LandingPageLinks.tsx:18]: This is very cool!

**Review by mwilkins74 — APPROVED** [review]

File: src/components/organisms/landingPageLinks/LandingPageLinks.tsx (hunk 5 above)
Comment (ChuckBTaylor) [reply src/components/organisms/landingPageLinks/LandingPageLinks.tsx:12]: Oh, yea. I was exploring the return. Left it in.

Comment (ChuckBTaylor) [reply src/components/organisms/landingPageLinks/LandingPageLinks.tsx:12]:
```suggestion

```

**Pull request merged**